app/webpack.common.js

```diff
@@ -38,6 +38,9 @@ function revParse(gitDir, ref) {
   return refMatch[1] || revParse(gitDir, refMatch[2])
 }
 
+let cliCommands = Fs.readdirSync(
```

Comment:
Actually, move this into a function in `script/dist-info.js` along with all our other variables.